script/nbb_tests.clj

```diff
@@ -107,7 +107,7 @@
                   "-e"
                   "(require '[honey.sql :as sql]) (sql/format {:select :foo :from :bar :where [:= :baz 2]})")))))
   (testing "add `:paths` from nbb.edn to classpath"
-    (nbb {:dir "test-scripts/paths-test"} "runner.cljs"))
+    (is (nbb {:dir "test-scripts/paths-test"} "runner.cljs")))
```

Comment:
What is the actual value returned here? A string right? Could we do an assertion on that?

Reply:
Right now it's returning `nil` but I should be able to modify the tested function to return a string